File: Src/Icm.Core/Localization/ILocalizationRepository.vb
Namespace Icm.Localization

    ''' <summary>
    '''  Methods to manage localized strings.
    ''' </summary>
    Public Interface ILocalizationRepository

        Default ReadOnly Property Item(key As String) As String
        ReadOnly Property ItemForCulture(lcid As Integer, key As String) As String

    End Interface
End Namespace

File: Src/Icm.Core/Localization/ILocalizationRepository.cs
namespace Icm.Localization
{

	/// <summary>
	///  Methods to manage localized strings.
	/// </summary>
	public interface ILocalizationRepository
	{


		string this[int lcid, string key] { get; }
	}
}

//=======================================================
//Service provided by Telerik (www.telerik.com)
//Conversion powered by NRefactory.
//Twitter: @telerik
//Facebook: facebook.com/telerik
//=======================================================
